pkg/api/validation/validation.go

```diff
@@ -1614,6 +1615,17 @@ func ValidateVolumeMounts(mounts []api.VolumeMount, volumes sets.String, fldPath
 		if mountpoints.Has(mnt.MountPath) {
```

Comment:
Side note: it looks to me like you would be able to have a VolumeMount containing `..` - do we want to allow that, @saad-ali?